pom.xml

```diff
@@ -116,15 +116,21 @@
 
         <dependency>
             <groupId>org.scalatest</groupId>
-            <artifactId>scalatest_2.11</artifactId>
-            <version>3.1.0</version>
+            <artifactId>scalatest_2.12</artifactId>
+            <version>3.0.5</version>
```

Comment:
Let's keep the version we were using (it was 3.1.0) even if we change the scala version.

Reply:
The build fails when I using the older scalatest version and scopt version. Hence the reason I changed to 

Caused by: java.lang.NoClassDefFoundError: scala/Product$class
        at org.scalatest.Outcome.<init>(Outcome.scala:47)
        at org.scalatest.Succeeded$.<init>(Outcome.scala:357)
        at org.scalatest.Succeeded$.<clinit>(Outcome.scala)
        at org.scalatest.Assertions$class.$init$(Assertions.scala:1186)
        at org.scalatest.flatspec.AnyFlatSpec.<init>(AnyFlatSpec.scala:1685)
        at org.wikimedia.analytics.hdfstools.TestHdfsRsyncExec.<init>(TestHdfsRsyncExec.scala:13)
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base/java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128)
        at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:341)
        at java.base/java.lang.Class.newInstance(Class.java:677)
        at org.apache.maven.surefire.junit.PojoTestSet.<init>(PojoTestSet.java:63)
        at org.apache.maven.surefire.junit.JUnit3Provider.createTestSet(JUnit3Provider.java:107)
        at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:97)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        ... 6 more

Reply:
Done